components/NotFoundPage.tsx

```diff
@@ -0,0 +1,17 @@
+import Link from "next/link";
+
+const NotFoundPage = () => {
+  return (
+    <div className="flex min-h-screen flex-col items-center  justify-center space-y-10 bg-dark-400 text-light-300 antialiased ">
+      <h3 className="text-3xl font-bold text-primary lg:text-5xl">404😅</h3>
+
+      <h3 className="text-xl lg:text-3xl">Look Like you&apos;re lost</h3>
+      <p className="text-secondary">the page you are looking for not available</p>
+      <Link href="/" className="text-white border border-primary p-4 hover:bg-primary">
+        Go back to home
```

Comment:
```suggestion
        Go back
```

Reply:
I've reviewed all the suggestions and have adjusted my code accordingly
![image](https://github.com/lucavallin/first-issue/assets/91373023/c26dd683-6a35-40d0-a772-2d2aef24272e)